src/tailwind.css

```diff
@@ -1,3 +1,12 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer utilities {
```

Comment:
this should be handled by our Tailwind utilities plugin -> `packages/utils/tailwind-plugins/utilities/index.js`